bin/mycroft-config

```diff
@@ -95,13 +95,13 @@ function validate_config_file() {
     return $result
 }
 
-_conf_file="~/.mycroft/mycroft.conf"
+_conf_file="~/.config/mycroft/mycroft.conf"
 function name_to_path() {
     case ${1} in
         "system")     _conf_file="/etc/mycroft/mycroft.conf" ;;
-        "user")       _conf_file=$(readlink -f ~/.mycroft/mycroft.conf) ;;
+        "user")       _conf_file=$(readlink -f ~/.config/mycroft/mycroft.conf) ;;
```

Comment:
should this use `$XDG_CONFIG_HOME` instead of the hard coded path?

Reply:
Yes it should. `${XDG_CONFIG_HOME:-$HOME/.config}`  to be exact.